### ISSUE TITLE
We couldn't get the server details from hetzners cloud server using hetzner cloud-api-net.

### ISSUE THREAD

**Issue by RajuSPR** [lk-code/hetzner-cloud-api-net#5]
We couldn't get the server details from hetzners cloud server using hetzner cloud-api-net. 

We are trying to get the servers since last week. But couldn't do it. Here I had attached the screenshots for your reference. I had followed by the below steps to get the server details. Finally, my request is terminated in one method without any response. Please give any suggestion or solution for me.

Step 1: 

![image](https://user-images.githubusercontent.com/48589501/55881325-13782480-5bc0-11e9-914f-1b11bdd7afce.png)

Step 2:
Above request is going to the GetAsync method in Server.cs. There is a call for SendRequest method in this method.

string responseContent = await ApiCore.SendRequest(url);

![image](https://user-images.githubusercontent.com/48589501/55882337-00665400-5bc2-11e9-9d22-16157ac266f0.png)

Step 3:

Finally, my request is terminated in the following highlighted code. 
![image](https://user-images.githubusercontent.com/48589501/55882670-8b474e80-5bc2-11e9-9d11-b3e5b29eb479.png)

Above method didn't return any values. the request goes for the GetStringAsync method, at this time request was terminated. Please help me. I'm in a critical situation. Please check and find the solution for my issue.

**Comment by lk-code** [comment]
Hi,
I will try to help. Which project type do you use? Xamarin, UWP, Android, iOS, Windows, etc.
Is there no exception?

**Comment by RajuSPR** [comment]
Hi,

I have using the web application( Asp .net C# ).

*Regards *
*Rajendran SP*

*Nimbylz Technology Solutions Pvt. Ltd.*

*Jr.Software Engineer*
Old No. 1, New No. 5, Pillayar Koil Street,

A1 Vijay Gurukrupa,

Jafferkhanpet, Chennai - 600 083. INDIA

Office – 91-044-24850074

WebSite: http://www.nimbylz.com

Email: rajendran.sp@nimbylz.com

------------------------------------------------------------------------------------------------------------------------------

*Disclaimer:* The information contained in this e-mail and attachments (if
any) are privileged and
confidential and are intended for the individual(s) or entity(ies) named in
this e-mail.
If you are not the intended recipient, or employee or agent, you are hereby
notified that dissemination,
distribution or copying of this communication or attachments thereof is
strictly prohibited.
IF YOU RECEIVED this communication in error, please immediately notify the
sender and DELETE
the original message from the Inbox.   Although the company has taken
reasonable precautions to ensure no viruses are present in this email, the
company cannot accept responsibility for any loss or damage arising from
the use of this email or attachments.

**Comment by RajuSPR** [comment]
Yes, that is correct. It didn't return any exception message.

*Regards *
*Rajendran SP*

*Nimbylz Technology Solutions Pvt. Ltd.*

*Jr.Software Engineer*
Old No. 1, New No. 5, Pillayar Koil Street,

A1 Vijay Gurukrupa,

Jafferkhanpet, Chennai - 600 083. INDIA

Office – 91-044-24850074

WebSite: http://www.nimbylz.com

Email: rajendran.sp@nimbylz.com

------------------------------------------------------------------------------------------------------------------------------

*Disclaimer:* The information contained in this e-mail and attachments (if
any) are privileged and
confidential and are intended for the individual(s) or entity(ies) named in
this e-mail.
If you are not the intended recipient, or employee or agent, you are hereby
notified that dissemination,
distribution or copying of this communication or attachments thereof is
strictly prohibited.
IF YOU RECEIVED this communication in error, please immediately notify the
sender and DELETE
the original message from the Inbox.   Although the company has taken
reasonable precautions to ensure no viruses are present in this email, the
company cannot accept responsibility for any loss or damage arising from
the use of this email or attachments.

**Comment by RajuSPR** [comment]
I have used a web application( Asp .net C# ).
Yes, that is correct. It didn't return any exception message.

**Comment by lk-code** [comment]
Your API token is correct and active? 😋 can you See any http response Insider the Methode? I Tested it yesterday and it works correct.

**Comment by RajuSPR** [comment]
My API token is correct. But how can I identify it's active or not.

**Comment by lk-code** [comment]
maybe another user has delete the key at the hetzner console 😉. can you send me the code in which you start the request?

**Comment by RajuSPR** [comment]
Yeah sure. Already I had attached the screenshot in the initial post. I'll send to you again.


        public static async Task server ()
        {
            lkcode.hetznercloudapi.Core.ApiCore.ApiToken = "Here My API Token"; 
            List<lkcode.hetznercloudapi.Api.Server> serverList = await lkcode.hetznercloudapi.Api.Server.GetAsync();
            
        }

**Comment by RajuSPR** [comment]
Please find out my issue and give me the solution.

**Comment by lk-code** [comment]
![Anmerkung 2019-04-11 231429](https://user-images.githubusercontent.com/1656571/55993544-aa6fda00-5caf-11e9-9739-bc5c5b96a502.jpg)

I just created a fresh ASP.NET project (classic - no .NET Core). Then the library is loaded via NuGet and the servers are loaded as in the screenshot. It worked.

**Comment by lk-code** [comment]
![Anmerkung 2019-04-11 231430](https://user-images.githubusercontent.com/1656571/55993803-48fc3b00-5cb0-11e9-9aa4-1f072e8992bd.jpg)

also with your method the servers are loaded correctly.

**Comment by lk-code** [comment]
![image](https://user-images.githubusercontent.com/1656571/55993977-c2942900-5cb0-11e9-92ec-16cd9c2db034.png)

I was thinking that maybe the API token is invalid, but then the Hetzner server returns an HTTP-401.

**Comment by lk-code** [comment]
can you send me your api token on a private way. Maybe there is something in your server response, which is not displayed with me. If I've checked that, you can delete the API token ;).

**Comment by RajuSPR** [comment]
Please share your personal email for send my api token.

**Comment by lk-code** [comment]
you can send it to xe77a@live.de

**Comment by RajuSPR** [comment]
Thanks a lot @lk-code . It's working fine. It's possible on a console application?

**Comment by lk-code** [comment]
I have not tested every project type, but the library is compiled as a .NET standard. So it should run in any supported project

**Comment by RajuSPR** [comment]
Thank you very much!......... I have one more doubt. How can I find out One server floating IPs, locations, Images, and Volumes?
For example, I need one server details with that server location details. How can I get this? Any conjunctions fields are available between servers and locations.

**Comment by RajuSPR** [comment]
Can you please help me.

**Comment by RajuSPR** [comment]
How can I get the server types, Datacenters, image details for the particular server using Hetzner cloud API?.

**Comment by RajuSPR** [comment]
I need clarification about the Hetzners cloud API. I have developed the new web application to access the servers using Hetzner cloud API. I had successfully gotten the following server details using hetzner cloud API (Name, Id, Status, Created date, IPv4, IPv6). But I need more server details. How can I get the server types, Datacenters, image details for the particular server using Hetzner cloud API?. Can you please help me.

![image](https://user-images.githubusercontent.com/48589501/56785866-8adec280-6814-11e9-8335-c7eeec60bb1b.png)
